types/src/shared/deployment.ts

```diff
@@ -0,0 +1,63 @@
+import * as child_process from "child_process";
+
+import { LoggerInterface } from "./logger";
+
+export async function sendDeploymentMessage({
+  service,
+  logger,
+}: {
+  service: string;
+  logger: LoggerInterface;
+}) {
+  const { SLACK_USER_OPERATION_BOT_TOKEN } = process.env;
+
+  if (!SLACK_USER_OPERATION_BOT_TOKEN) {
+    logger.info({}, "SLACK_USER_OPERATION_BOT_TOKEN is not set");
+    return;
+  }
+
+  // get the current commit id
+  let commitId = "unknown";
+
+  try {
+    commitId = child_process.execSync("agit rev-parse HEAD").toString().trim();
```

Comment:
agit ?

Reply:
this was to test what happens when git fails :/ Thanks for catching it.